### ISSUE TITLE
Aegisub crashes when time format is set to ja_JP on KDE Plasma

### ISSUE THREAD

**Issue by Ericchi** [Aegisub/Aegisub#185]
I am currently using KDE Plasma on Manjaro Linux.
I had my regional settings set to the US region format and the Time/date set to the Japanese region format.
This for some reason causes Aegisub to crash at launch without explanation nor error codes.

Here's an easy (and I hope temporary) fix. Launch Aegisub with the following command:
env LC_TIME=en_US.UTF-8 GDK_BACKEND=x11 aegisub-3.2 %f

**Comment by wangqr** [comment]
Might be similar to #172 . Can you make sure your locale settings is available on your system? You can get a list of available locales by running `locale -a`.

If it looks normal, I recommend running Aegisub in gdb or strace to get some debug output.

**Comment by Ericchi** [comment]
> Might be similar to #172 . Can you make sure your locale settings is available on your system? You can get a list of available locales by running `locale -a`.
> 
> If it looks normal, I recommend running Aegisub in gdb or strace to get some debug output.

You are right. The locale wasn't available. After adding the Japanese locale it runs fine without having to edit the launcher's command.
